src/main/java/org/primefaces/component/export/DataExporter.java

```diff
@@ -64,11 +66,13 @@
     private ValueExpression exporter;
 
     public DataExporter() {
+        ResourceUtils.addComponentResource(FacesContext.getCurrentInstance(), "filedownload/filedownload.js");
```

Comment:
Didn't know if there was a better way to include this?  I guess there is no renderer to attach to so I guess this is OK.

Reply:
No renderer. Don´t know a better place. It´s the same FileDownloadActionListener does.